documentation/specs/src/economics/public-goods-funding.md

```diff
@@ -198,44 +200,36 @@ impl addRecipient for cPgfRecipients
 impl remRecipient for cPgfRecipients
 ```
 
-### End of Term Summary
+## PGF stewards incentives
 
-At the end of each term, the council is encouraged to submit a "summary"  which describes the funding decisions the councils have made and their reasoning for these decisions. This summary will act as an assessment of the council and will be the primary document on the basis of which governance should decide whether to re-elect the council.
+Being a PGF steward is (should be) hard work. Stewards must invest their time in crafting proposals, being in touch with the opinions of the Namada community, and taking the risks associated with being voted out. In order to incentivise this effort, Namada allocates a set amount of NAM inflation that is directed towards the Stewards. This parameter is set by governance and is included in genesis by default. That allocation is then split equally between each PGF Steward.
```

Comment:
```suggestion
Being a PGF steward is (should be) hard work. Stewards must invest their time in crafting proposals, being in touch with the opinions of the Namada community, and taking the risks associated with being voted out. In order to incentivise this effort, Namada allocates a set amount of NAM inflation that is directed towards the stewards. This parameter is set by governance and is included in genesis by default. That allocation is paid to each steward.
```

we do not want to split it between them, it creates competitive dynamics. there should just be a fixed rate that is paid to every steward (so more is paid if there are more stewards, and less is paid if there are fewer stewards)